Queueomatic/Queueomatic.DataAccess/Repositories/Interfaces/IRoomRepository.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Queueomatic.DataAccess.Models;
+
+namespace Queueomatic.DataAccess.Repositories.Interfaces;
+
+public interface IRoomRepository : IRepository<Room, int> 
```

Comment:
Yes, I agree. However, only for the room type as that is the only model which uses hashids. Otherwise we will simply send the content as JSON and let the serializer deal with the parsing.

Reply:
Sounds good. But we can change that in a separate issue,
I close this as it is.

Reply:
On second thought, this should not be handled in the repositories. We should use the datamodels which most closely resembles our database and if we need to send specific data back to the client it should be done in the handlers. The same applies if the user sends data. We decode in the handler and then query the database with the correct data.

The repositories should be as pure as possible. We do not want to put data transfer objects there. That's for the handlers or even the client implement.

Reply:
I got your point. But would you say that it's not a domain layers job?

I think that it's part of the domain layers responsibility that handle these kinds of logic and the server does not need to know the real objects. By using the DTO we can assure that this is not going to happen even if it "accidentally" looks the same.